website/client-old/js/services/taskServices.js

```diff
@@ -72,10 +72,16 @@ angular.module('habitrpg')
     };
 
     function updateTask (taskId, taskDetails) {
+      var taskDetailsToSend = _.omit(
+        taskDetails,
+        ['challenge', 'completed', 'createdAt', 'group', 'history', 'id',
```

Comment:
I'm just going to make that adjustment and merge this in. We can change it later if necessary.